### ISSUE TITLE
How to run on local files?

### ISSUE THREAD

**Issue by Erik262** [m1guelpf/yt-whisper#25]
Hi, nice project!!
Do you know how to run this on local .mp4 files? Is there a command I could use? Nothing was mentioned in your readme.

Thanks in advance! :)

**Comment by rhorer** [comment]
Yes, I would like to be able to run this on local files as well.

**Comment by sloganking** [comment]
I may implement this in a pull request soon. I have a PoC working on my machine. I just haven't cleaned it up. A question for those here. Would you prefer to just replace the URL passed with a file directory, and the script will just figure out which one it is? Or do you want to have separate flags for passing a URL or a directory?

I vote for it just figuring it out.

**Comment by Erik262** [comment]
Same here. Vote for figuring out.

**Comment by msjsc001** [comment]
It would be nice to be able to translate via local paths
